content/en/blog/2020/wasmhub-istio/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebAssembly to Envoy and Istio
+subtitle: A place to build, publish, share, and deploy WebAssembly Envoy extensions
+description: Community partner tooling of Wasm for Istio by Solo.io.
+publishdate: 2020-03-13
+attribution: "Idit Levine (Solo.io)"
+keywords: [wasm,extensibility,alpha,performance,operator]
+---
+
+*[Originally posted on the Solo.io blog](https://www.solo.io/blog/an-extended-and-improved-webassembly-hub-to-helps-bring-the-power-of-webassembly-to-envoy-and-istio/)*
```

Comment:
Are you sure it makes sense to just repost an already posted blog? There are already two other istio.io posts on wasm, including the wasm hub one from Christian Posta.